.github/workflows/pull-request.yml

```diff
@@ -250,13 +250,13 @@ jobs:
 
       - name: Filter files
         id: get-lint-files
-        run: node ./script/github-actions/filter-files-changed.js ${{ steps.get-files-changed.outputs.files_modified }}
+        run: node ./script/github-actions/filter-files-changed.js ${{ steps.get-files-changed.outputs.files }}
 
       - name: Annotate ESLint results
         run: yarn run eslint -c ./.eslintrc.changed.js --ext .js --ext .jsx --no-error-on-unmatched-pattern --format ./script/github-actions/eslint-annotation-format.js ${{ steps.get-lint-files.outputs.JSFILES }}
 
       - name: Run Stylelint
-        if: ${{ steps.get-lint-files.outputs.SCSSFILES != '' }}
+        if: ${{ always() && steps.get-lint-files.outputs.SCSSFILES != '' }}
```

Comment:
Since we're always running `Annotate Stylelint results` we should probably always be running this step